code/modules/detectivework/tools/rag.dm

```diff
@@ -228,10 +228,10 @@
 		return
 
 	//also copied from matches
-	if(REAGENT_VOLUME(reagents, /singleton/reagent/toxin/phoron)) // the phoron explodes when exposed to fire
+	if(REAGENT_VOLUME(reagents, /singleton/reagent/toxin/phoron/base)) // the phoron explodes when exposed to fire
```

Comment:
Surely this shouldn't be checking specifically for normal phoron? All phoron should be flammable/dangerous

Reply:
If all phoron is flammable why is k'ois phoron useless as fuel